admin/views.py

```diff
@@ -553,10 +577,17 @@ def resource_edit(resource_type, resource_id):
 
     for attribute in editable_attributes:
         attribute_value = request.form.get(attribute["name"])
-        print('attribute_value....', attribute_value)
-        validated_attribute_value = validate_resource_attribute(resource_type, attribute, attribute_value)
-        print('validated_attribute_value....', validated_attribute_value)
-        setattr(resource, attribute["name"], validated_attribute_value)
+        validated_attribute_value = validate_resource_attribute(
+            resource_type, attribute, attribute_value
+        )
+
+        if attribute["name"] == admin_configs["user"]["secret"]:
+            hashed_password = bcrypt.generate_password_hash(
```

Comment:
How this password is showing up for user on the edit screen or on create screen?

Reply:
@kuldeep3 It is showing in hash form.